test/rules/cascade_invocations.dart

```diff
@@ -194,3 +194,40 @@ void function694() {
   Bug694.bar = 2;
   Bug694.foo = 3; // OK
 }
+
+class A {
+  int f;
+  int g;
+
+  void x() {}
```

Comment:
You don't seem to be using these methods. If so, remove them.

Reply:
Done. Good catch.